src/components/Appointments/AppointmentBookingConfirmation.vue

```diff
@@ -24,15 +24,15 @@
 	<div class="appointment-booking-confirmation">
 		<EmptyContent :title="$t('calendar', 'Please confirm your reservation')" :description="$t('calendar', 'We sent you an email with details. Please confirm your appointment using the link in the email. You can close this page now.')">
 			<template #icon>
-				<CheckIcon decorative />
+				<EmailFastIcon decorative />
 			</template>
 		</EmptyContent>
 	</div>
 </template>
 
 <script>
 import EmptyContent from '@nextcloud/vue/dist/Components/NcEmptyContent.js'
-import CheckIcon from 'vue-material-design-icons/Check.vue'
+import EmailFastIcon from 'vue-material-design-icons/EmailFast.vue'
```

Comment:
**issue:** Designers prefer icons from [the official Google Material set](https://fonts.google.com/icons?icon.query=email), not [the MDI project](https://pictogrammers.com/library/mdi/icon/email-fast/).

Reply:
Sure, that seems to make sense to me, but I don't know if that's a valid comment for here. A quick search for 'vue-material-design-icons' in the codebase shows 100+ results for this iconset in use. If the material set should be changed, then it should be done all at once:
<img width="482" alt="Screen Region 2023-05-15 at 21 50 17" src="https://github.com/nextcloud/calendar/assets/150431/5323dcb9-2ae5-4694-83dc-cbb3bc4b5e9a">

What this fix is trying to achieve is different. The task of changing icon set is outside the scope of this PR, and should be separate IMO.

Reply:
Sorry I wasn't clear. I didn't ask to change the icon set, we use that everywhere, I asked for the icon to be picked in the official set of Google Material Icons, not from the community ones from the MDI library. These ones are less stable, [some are removed from time to time](https://pictogrammers.com/docs/library/mdi/releases/changelog/).

Reply:
Ah I see, that makes more sense! I have changed the icon to the 'EmailMarkAsUnreadIcon' icon. However, the actual icon is differnet from the link you sent here: https://fonts.google.com/icons?icon.query=email&selected=Material+Symbols+Outlined:outgoing_mail:FILL@0;wght@400;GRAD@0;opsz@48

Also, tbh I don't think it's as clear as the EmailFastIcon, but it should work. Here are some other options:

EmailMarkAsUnreadIcon
<img width="585" alt="Screen Region 2023-05-15 at 22 11 50" src="https://github.com/nextcloud/calendar/assets/150431/906b2504-6d58-4827-b54a-f73d6bf73483">

EmailIcon
<img width="597" alt="Screen Region 2023-05-15 at 22 11 40" src="https://github.com/nextcloud/calendar/assets/150431/87056077-f4d9-4d50-b308-3f40f1f383bf">

EmailArrowRightOutline
<img width="561" alt="Screen Region 2023-05-15 at 22 11 28" src="https://github.com/nextcloud/calendar/assets/150431/972441c4-3c11-45e3-8a1a-91b827a96063">

EmailFastIcon
<img width="590" alt="Screen Region 2023-05-15 at 20 51 49" src="https://github.com/nextcloud/calendar/assets/150431/dab5e1fa-9fd5-4ea6-9253-1102d7ebd0a7">

Reply:
Happy to help! I tried to use that icon (mark_email_unread), but it doesn't exist in the 'vue-material-design-icons' project. Here are all the icons that exist: https://www.npmjs.com/package/vue-material-design-icons?activeTab=code

The closest was 'EmailMarkAsUnreadIcon' which if you look at the example above, it doesn't look right/great. Do you have a preferred icon - or how can I get that exact 'mark_email_unread'?

Thanks for the link to the design page - I will definitely take a look! Hopefully I can add a bit more on this app to polish the 'Appointments' feature. It's really powerful but needs just a few small tweaks.

Thanks to you and the team for all the hard work :)

Reply:
@pjrobertson ah ok, strange. Anyway, in that case I think the simple "EmailIcon" is probably best. :)

And yes, looking forward for further contributions from you!

Reply:
Alright - I updated to the EmailIcon and pushed the changes. Looking forward to seeing this in the next release. Thanks! :)